src/__tests__/inventory/AddInventoryItem.test.tsx

```diff
@@ -1,9 +1,98 @@
+import React from 'react';
+
+import createMockStore from 'redux-mock-store'
+import thunk from 'redux-thunk'
+import { fireEvent, render, screen, waitFor } from '@testing-library/react'
+import { Router, Route } from 'react-router-dom'
+
+import Inventory from '../../inventory/Inventory';
+import { TitleProvider } from '../../page-header/title/TitleContext';
+import { RootState } from '../../shared/store'
+import Permissions from '../../shared/model/Permissions'
+import { createMemoryHistory } from 'history'
+import { Provider } from 'react-redux';
+
+const mockStore = createMockStore<RootState, any>([thunk])
+
+const setup = (
+  route: string,
+  permissions: Permissions[],
+  path: string
+) => {
+  const history = createMemoryHistory({ initialEntries: [route] })
+  const store = mockStore({ user: { permissions } } as any)
+  return {
+    history, ...render(
+      <Provider store={store}>
+        <Router history={history}>
+          <Route path={path}>
+            <TitleProvider>
+              <Inventory />
+            </TitleProvider>
+          </Route>
+        </Router>
+      </Provider>
+    )
+  }
+}
+
 describe('AddInventoryItem', () => {
-  it('add item and cancel buttons render', () => {})
+  it('add item and cancel buttons render', () => {
+    /* 1. Render the component 
+     * 2. Check if the add item button has been rendered
+     * 3. Then, check if the cancel item button renders
+     */
```

Comment:
Since this comment is pseudocode for our own purposes, it should be okay to remove.